src/components/menu/menu-full-screen/menu-full-screen.spec.tsx

```diff
@@ -354,6 +355,28 @@ describe("MenuFullscreen", () => {
       { pt: "10px", pb: "10px" },
       (component) => component.find(StyledMenuItem)
     );
+
+    testStyledSystemPadding(
+      (props) => (
+        <MenuContext.Provider
+          value={{
+            menuType: "light",
+            openSubmenuId: null,
+            inFullscreenView: true,
+            inMenu: true,
+            setOpenSubmenuId: () => {},
+          }}
+        >
+          <MenuItem submenu="foo">
+            <MenuItem {...props} href="#">
+              bar
+            </MenuItem>
+          </MenuItem>
+        </MenuContext.Provider>
+      ),
+      undefined,
+      (component) => component.find(StyledSubmenu).find(StyledMenuItem)
+    );
   });
```

Comment:
**comment(minor)**: I realise we want to avoid unnecessary wrapping, but what do you think about describing which component's padding is being tested so its clear when ran in the terminal?


```suggestion
  describe("MenuItem padding", () => {
    testStyledSystemPadding(
        (props) => (
          <MenuContext.Provider
            value={{
              menuType: "light",
              openSubmenuId: null,
              inFullscreenView: true,
              inMenu: true,
              setOpenSubmenuId: () => {},
            }}
          >
            <MenuItem submenu="foo">
              <MenuItem {...props} href="#">
                bar
              </MenuItem>
            </MenuItem>
          </MenuContext.Provider>
        ),
        undefined,
        (component) => component.find(StyledSubmenu).find(StyledMenuItem)
      );
    });
  });
```

Reply:
Good idea I'll see if I can add something to make it a bit easier on the eye without convoluting the log output or nesting excessively.